xml/apache2.xml

```diff
@@ -2161,8 +2161,8 @@ sudo cp new.cert.key /etc/apache2/ssl.key/server.key</screen>
      have an officially signed certificate.
     </para>
     <para>
-     The best-known commercial CAs are Comodo
-     (<link xlink:href="https://www.comodo.com"/>) and Symantec
+     The best-known commercial CAs are Thawte
```

Comment:
Thawte is now Symantec (even their logo is "Thawte from Symantec" now). Also, Comodo is now the largest CA, I think. How about adding GoDaddy or DigiCert too?